code/modules/admin/admin_verbs.dm

```diff
@@ -219,7 +219,7 @@ GLOBAL_PROTECT(admin_verbs_debug)
 	/client/proc/reload_configuration,
 	/client/proc/restart_controller,
 	/client/proc/run_empty_query,
-	/client/proc/SDQL2_query,
+	/client/proc/admin_SDQL2_query,
```

Comment:
alphabetizing this list screws with admin verb ordering which is another can of worms